qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -1,6 +1,6 @@
 import com.google.protobuf.util.JsonFormat
+import groups.COMPATIBILITY
```

Comment:
this isn't used here

Reply:
Good catches, I had added tests from these two files to COMPATIBILITY then removed them again later.